ordered_model/models.py

```diff
@@ -88,7 +89,7 @@ def increase_order(self, **extra_kwargs):
             update_kwargs.update(extra_kwargs)
         return self.update(**update_kwargs)
 
-    def bulk_create(self, objs, batch_size=None):
+    def bulk_create(self, objs, batch_size=None, ignore_conflicts=False):
```

Comment:
Would it be better to use `def bulk_create(self, *args, **kwargs):` as we are not using any of the arguments and pass all to the parent class?
There would be no version check be required, if we are not using specific arguments.

```
     def bulk_create(self, objs, *args, **kwargs): 
```